### DISCUSSION TITLE
Proposal for community building engagement and management.

### DISCUSSION THREAD

**Discussion by Michaelson11** [community/community#33895] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm a professional engagement officer and experienced CM.

As an engagement officer I have a team of angels who work to make your community more lively and educative.

We do this by engaging in conversations about the features and benefits of the project.

This helps attract more members who can translate to investors and our service charge is quite affordable.

You'd agree with me that people are often attracted to active communities as it portrays the success possibility of the project.

For the community engagement / content seeding team.

We run from 100 messages upto 1000 messages daily, depending on your choice.

For moderating I take $100 per week.{ Negotiable}

 

I also have a team of professional shillers

We do Dex listing

Content creation

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Michaelson11 , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions so we'll be locking then removing this post. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
